src/Robo.php

```diff
@@ -298,6 +306,8 @@ public static function addInflectors($container)
             ->invokeMethod('setHookManager', ['hookManager']);
         $container->inflector(\Robo\Contract\VerbosityThresholdInterface::class)
             ->invokeMethod('setOutputAdapter', ['outputAdapter']);
+        $container->inflector(\Robo\Contract\ClassLoaderAwareInterface::class)
+            ->invokeMethod('setClassLoader', ['classLoader']);
```

Comment:
Do we really need to inflect this? Inflection allows any CommandFile (et. al.) to have access to the class loader. If we only need global (built-in) classes to do this, typically, then we should probably just inject this reference during container initialization.